examples/Demo/FluentUI.Demo.Client/Documentation/Components/Checkbox/Examples/CheckboxDisabled.razor

```diff
@@ -0,0 +1,7 @@
+﻿<div style="display: flex;flex-direction: column; gap: 12px;">
+    <FluentCheckbox Shape="@CheckboxShape.Square" Label="Square disabled checked" Disabled="true" Checked="true" />
+    <FluentCheckbox Shape="@CheckboxShape.Circular" Label="Circular disabled checked" Disabled="true" Checked="true" />
+
+    <FluentCheckbox Shape="@CheckboxShape.Square" Label="Square disabled unchecked" Disabled="true" Checked="false" />
```

Comment:
Please don't display all possibilities: check/uncheck... The user can click on it to display the different views. So only a `Square` and a `Circular` checkbox.

Reply:
Fixed in the next commit